conf/server/server-mysql.conf

```diff
@@ -0,0 +1,55 @@
+server {
+    bind_address = "127.0.0.1"
+    bind_port = "8081"
+    registration_uds_path = "/tmp/spire-registration.sock"
+    trust_domain = "example.org"
+    data_dir = "./.data"
+    log_level = "DEBUG"
+    upstream_bundle = true
+    svid_ttl = "1h"
+    ca_subject = {
+        Country = ["US"],
+        Organization = ["SPIFFE"],
+        CommonName = "",
+    }
+}
+
+/*
+Development database set-up only, NOT PRODUCTION:
+```
+$ mysql -u root
+mysql> CREATE DATABASE spire_development;
+mysql> CREATE USER 'spire'@'localhost';
+mysql> GRANT ALL PRIVILEGES ON spire_development.* TO 'spire'@'localhost'
+mysql> FLUSH PRIVILEGES;
+```
+*/
+plugins {
+    DataStore "sql" {
+        plugin_data {
+            database_type = "mysql"
+            connection_string = "spire:@tcp(127.0.0.1)/spire_development?parseTime=true"
```

Comment:
From what I can tell, `parseTime` is required for the plugin to operate properly. Configure() should probably fail if `parseTime` isn't set to `true`, or alternatively parse the connection string and unconditionally add it if missing.

Reply:
Agreed. I can add a validation for that.